lib/index.ts

```diff
@@ -9,15 +9,15 @@ export default {
     return require('./server');
   },
   get getMiddleware() {
-    return require('./middleware');
+    return require('./middleware').default;
```

Comment:
this looks like a breaking change, if our own code must use `.default` what if your consumers require?

Reply:
Fixed, this was unintended. I also confirmed locally that what is being exported is the same